holoviews/plotting/bokeh/renderer.py

```diff
@@ -121,6 +121,8 @@ def get_plot(self_or_cls, obj, doc=None, renderer=None):
         doc = curdoc() if doc is None else doc
         if self_or_cls.theme:
             doc.theme = self_or_cls.theme
+        elif self_or_cls.theme is None:
+            doc.theme = None
```

Comment:
Would it be easier just to do:

```
doc.theme = self_or_cls.theme
```

BokehRenderer.theme can only be an Theme instance or None.